homeassistant/components/isy994/services.py

```diff
@@ -116,6 +122,16 @@ def valid_isy_commands(value: Any) -> str:
     vol.Required(CONF_COMMAND): vol.In(VALID_NODE_COMMANDS)
 }
 
+SERVICE_RENAME_NODE_SCHEMA = {vol.Required(CONF_NAME): cv.string}
+
+SERVICE_GET_ZWAVE_PARAMETER_SCHEMA = {vol.Required(CONF_PARAMETER): vol.Coerce(int)}
+
+SERVICE_SET_ZWAVE_PARAMETER_SCHEMA = {
+    vol.Required(CONF_PARAMETER): vol.Coerce(int),
+    vol.Required(CONF_VALUE): vol.Coerce(int),
+    vol.Required(CONF_SIZE): vol.In(VALID_PARAMETER_SIZES),
```

Comment:
```suggestion
    vol.Required(CONF_SIZE): vol.All(vol.Coerce(int), vol.In([VALID_PARAMETER_SIZES])),
```

Reply:
Pretty sure this will work, but I don't have zwave to validate it

Reply:
Updated. Can't verify at the moment either, but tested that the service call passes the schema validation. The backend on pyisy can handle either integer or string and casts as string before calling the resource anyways--don't see any reason why it wouldn't work.